test/functional/mempool_accept.py

```diff
@@ -139,6 +122,7 @@ def run_test(self):
         self.log.info('A transaction that conflicts with an unconfirmed tx')
         # Send the transaction that replaces the mempool transaction and opts out of replaceability
         node.sendrawtransaction(hexstring=tx.serialize().hex(), maxfeerate=0)
+        #self.mempool_size += 1
```

Comment:
I think this line should be deleted. If you meant it communicate something about the test, please clarify